.github/workflows/ci.yml

```
@@ -0,0 +1,150 @@
+          key: ${{ runner.os }}-bun-${{ hashFiles('car_fleet_manager_frontend/requirements.txt') }}
+
+      - name: Setup bun (official action)
+        uses: oven-sh/setup-bun@v1
+        with:
+          version: '1'
+
+      - name: Build Reflex frontend
+        working-directory: ./car_fleet_manager_frontend
+        run: |
+          python -m reflex build
+name: CI
```

actionlint (1.7.7)

key "name" is duplicated in workflow. previously defined at line:1,col:1

(syntax-check)